index.html

```diff
@@ -377,6 +449,14 @@ <h3>Defining Zome Functions</h3>
           </fragment>
         </section>
 
+        <section>
+          <fragment language="markdown" animate="by-line with-ancestry">
+            ### Entry Retrievals
+            #### Overview
+
+            - Read requests issued to the DHT
+          </fragment>
+
```

Comment:
We are missing a closing "</section>" here.

Reply:
I'm not seeing it... Where was it opened? @mmeigooni if you see it could you fix it?

Reply:
I'm not seeing it... Where was it opened? @mmeigooni if you see it could you fix it?

Reply:
@guillemcordoba I was not able to locate the issue you're referring to here. Have to merge this for tomorrow. Please push an edit when you get a moment?